process/merge_requests/README.md

```diff
@@ -0,0 +1,104 @@
+# Merge requests
+## Introduction
+Since introducing Gitlab to our workflow, we've been using it as our primary
+tool for reviewing code.  
+While issues still reside in Redmine and the actual merging/deploying is handled
+outside of Gitlab, commenting and discussing code is handled through Gitlab.
+
+While there has been discussion about altering the workflow to utilize Gitlab's
+various features more extensively, for now you should only consider it being
+something that rests on top of your everyday work in Redmine.
+
+## Prerequisite
+The following instructions assume you have a working Gitlab account. If you
+don't yet have access to the web interface of Gitlab, contact Operations.
+
+## Creation procedure
+Let's assume you've worked on an issue enough to set the status to "Feedback",
+which would put it into the Feedback-queue.
+
+Before you set the status to "Feedback", make sure that there's a Merge Request
+(henceforth referred to as "MR") available in Gitlab.
+
+These are the steps necessary:
+
+1. Log into [Gitlab](https://gitlab.bgops.net/).
+2. When logged in, you should arrive at the Dashboard (as written in the top
+   left). Click "Projects" in the menu to the left.
+3. Choose the project which you wish to submit a MR for. Most of the times, this
+   will be "sparkle".
+4. Click "Merge Requests" in the menu to the left.
+5. Click the green "New Merge Request" button, located in the top right part of
+   the page.
+6. Here you'll need to provide the **source** branch and the **target** branch.
+   The source branch should be the branch which you wish to have review, while
+   the target branch should be what you want to merge into.  
+   Unless you've been asked to set it to something else, you'll want the target
+   branch to be set to `master`.
+7. Press the blue "Compare branches" button.
+8. Enter title and description, following the guidelines written in the
+   **Title and Description guidelines** section.  
+   Set no label, this will be done by someone else.  
+   Set no assignee, unless you've been asked to.
+9. Press the "Submit merge request" button.
```

Comment:
Personally, I would want to see that included. However, I'm not sure if including this kind of directive would lead to apprehensiveness against submitting the MRs. I'd like to hear what others think first.